### PR TITLE
Copy Charlie's page and format for docsify

### PR DESCRIPTION
**Pull request by ErikMHummel**

### CODE REVIEW

**Review by singhald — APPROVED** [review]

File: lightning-tag/implementation-setup-tasks.md [hunk 1]
```diff
@@ -1,5 +1,28 @@
...
+Before deploying the Lotame Lightning Tag on your site, you must establish the following settings within the DMP:
 
-[comment]: # (Charlie is working on steps & screenshots.)
+1. **Domain Whitelist** - the list of domains on which Lightning Tag is allowed to collect data and utilize third-party cookies, first-party cookies, and localStorage.
+1. **Profile Extraction: Audience Owner Client** - The client id for which to provide audiences.
+1. **Profile Extraction: Audience Format** - Decide whether Lightning Tag will provide audiences as ids or targeting codes.
+
+## Domain Whitelisting
+
+For Lightning Tag to operate on your site, the domain must first be whitelisted. Domains can be added individually, or in bulk, directly into the DMP by navigating to ‘Client Admin: Manage Settings’ → ‘Domains.’
```
Comment (noahpc) [inline lightning-tag/implementation-setup-tasks.md]: * Simpler wording: Domains can be added in the DMP Platform UI by navigating to 'Client Admin.........

File: lightning-tag/implementation-setup-tasks.md [hunk 2]
```diff
@@ -1,5 +1,28 @@
...
+
+By default, Lightning Tag returns audiences owned by the client ID passed to the tag. In many client use-cases, audiences are created and managed within a separate parent account. If this is your use-case, your Lotame account manager must adjust the settings to return audiences from this linked parent account.
+
+Please work with your Lotame account manager or support@lotame.com to ensure the correct settings are established.
+
+## Profile Extraction: Audience Format
+
+For the audiences included in the callback, Lotame can either return the audience ID or ‘targeting code’ (a custom value assigned to each audience).
+
+- Audience Id: `{"pid":"9132c153ba0233f7462881b2843e0932","tc":["12345","26654"]}`
```
Comment (noahpc) [inline lightning-tag/implementation-setup-tasks.md]: This references our internal /6 call - this should instead reference the profile callback getAudiences() function

File: lightning-tag/implementation-setup-tasks.md [hunk 3]
```diff
@@ -1,5 +1,28 @@
 # Pre-Implementation Setup
 
-!> Coming soon.
+Before deploying the Lotame Lightning Tag on your site, you must establish the following settings within the DMP:
 
-[comment]: # (Charlie is working on steps & screenshots.)
+1. **Domain Whitelist** - the list of domains on which Lightning Tag is allowed to collect data and utilize third-party cookies, first-party cookies, and localStorage.
```
Comment (ctmitchell28) [inline lightning-tag/implementation-setup-tasks.md]: The list

File: lightning-tag/implementation-setup-tasks.md [hunk 4]
```diff
@@ -1,5 +1,28 @@
...
+1. **Profile Extraction: Audience Owner Client** - The client id for which to provide audiences.
+1. **Profile Extraction: Audience Format** - Decide whether Lightning Tag will provide audiences as ids or targeting codes.
```
Comment (ctmitchell28) [inline lightning-tag/implementation-setup-tasks.md]: Whether Lightning Tag will return audiences as ids or targeting codes

File: lightning-tag/implementation-setup-tasks.md (hunk 1 above)
Comment (ctmitchell28) [inline lightning-tag/implementation-setup-tasks.md]: For Lightning Tag to operate on your site, the domain must first be whitelisted. Domains can be whitelisted individually, or in bulk, directly within your DMP account by navigating to ‘Client Admin: Manage Settings’ → ‘Domains.’

File: lightning-tag/implementation-setup-tasks.md [hunk 5]
```diff
@@ -1,5 +1,28 @@
...
+
+## Domain Whitelisting
+
+For Lightning Tag to operate on your site, the domain must first be whitelisted. Domains can be added individually, or in bulk, directly into the DMP by navigating to ‘Client Admin: Manage Settings’ → ‘Domains.’
+
+?> Ensure the domain is whitelisted to the correct client account. You may have access to multiple linked child accounts under your main parent account. The domain should be added to the client ID that is being applied to the tag on your web pages.
+
+## Profile Extraction: Audience Owner Client
+
+By default, Lightning Tag returns audiences owned by the client ID passed to the tag. In many client use-cases, audiences are created and managed within a separate parent account. If this is your use-case, your Lotame account manager must adjust the settings to return audiences from this linked parent account.
```
Comment (ctmitchell28) [inline lightning-tag/implementation-setup-tasks.md]: By default, Lightning Tag returns audiences owned by the client ID passed to the tag. In many cases, audiences are created and managed within a separate parent account. If this is how you are set up within the DMP, your Lotame account manager must adjust the settings to return audiences from this linked parent account.

File: lightning-tag/implementation-setup-tasks.md (hunk 2 above)
Comment (ctmitchell28) [inline lightning-tag/implementation-setup-tasks.md]: Do I have the parameter right, or is it 'aud' for audience ids?

Comment (ErikMHummel) [reply lightning-tag/implementation-setup-tasks.md]: They don't get that full string. Lightning Tag has functions to pull the values individually so there is no parameter names. 

https://lotame.github.io/docs/#/lightning-tag/detailed-reference?id=audience-callback

So yes, this should change to discuss the difference in output from calling `getAudiences` which is described on the above page as:

>Array of either targeting codes as strings or audience ids as integers, based on your established tag settings. Example: ["aud1","aud2"] or [123,789]

File: lightning-tag/implementation-setup-tasks.md (hunk 3 above)
Comment (ErikMHummel) [reply lightning-tag/implementation-setup-tasks.md]: Will fix shortly.

File: lightning-tag/implementation-setup-tasks.md (hunk 4 above)
Comment (ErikMHummel) [reply lightning-tag/implementation-setup-tasks.md]: Will fix shortly.

File: lightning-tag/implementation-setup-tasks.md (hunk 1 above)
Comment (ErikMHummel) [reply lightning-tag/implementation-setup-tasks.md]: Will fix shortly.

File: lightning-tag/implementation-setup-tasks.md (hunk 5 above)
Comment (ErikMHummel) [reply lightning-tag/implementation-setup-tasks.md]: Will fix shortly.

**Review by ctmitchell28 — APPROVED** [review]

File: lightning-tag/implementation-setup-tasks.md [hunk 6]
```diff
@@ -1,5 +1,28 @@
...
+Before deploying the Lotame Lightning Tag on your site, you must establish the following settings within the DMP:
 
-[comment]: # (Charlie is working on steps & screenshots.)
+1. **Domain Whitelist** - The list of domains on which Lightning Tag is allowed to collect data and utilize third-party cookies, first-party cookies, and localStorage.
+1. **Profile Extraction: Audience Owner Client** - The client id for which to provide audiences.
+1. **Profile Extraction: Audience Format** - Whether Lightning Tag will return audiences as ids or targeting codes.
+
+## Domain Whitelisting
+
+For Lightning Tag to operate on your site, the domain must first be whitelisted. Domains can be whitelisted individually, or in bulk, directly within your DMP account by navigating to ‘Client Admin: Manage Settings’ → ‘Domains.’.
```
Comment (noahpc) [inline lightning-tag/implementation-setup-tasks.md:11]: two periods at end of this sentence

**Review by noahpc — APPROVED** [review]

**Pull request merged**